the-9-exercises/src/App.jsx

```diff
@@ -1,14 +1,9 @@
-
-import './App.css'
-
+import Fetch from "./components/Fetch";
 function App() {
-
-
   return (
     <>
-      <h1 className='bg-red-400'>hey there</h1>
+      <Fetch />
     </>
-  )
+  );
 }
```

Comment:
If you are returning one component you don't need to wrap it inside fragments again you can return it directly.